src/Files.App/Utils/Storage/Operations/FileOperationsHelpers.cs

```diff
@@ -1018,5 +1020,26 @@ protected override void Dispose(bool disposing)
 				}
 			}
 		}
+
+		public static string GetIncrementalName(bool overWriteOnCopy, string? filePathToCheck, string? filePathToCopy)
```

Comment:
```suggestion
		private static string GetIncrementalName(bool overWriteOnCopy, string? filePathToCheck, string? filePathToCopy)
```
Sorry, I forgot one comment.
It should be `private` if it doesn't need to be public.